xbmc/pvr/epg/EpgTagsContainer.cpp

```diff
@@ -643,10 +643,15 @@ void CPVREpgTagsContainer::QueuePersistQuery()
   }
 }
 
-void CPVREpgTagsContainer::Delete()
+void CPVREpgTagsContainer::Delete(bool bCommit)
```

Comment:
This method will not be called with bCommit= true and thus impl. can be simplified.